OptimizelySDK/Optimizely.cs

```diff
@@ -1362,6 +1362,17 @@ internal void IdentifyUser(string userId)
             Dictionary<string, object> data
         )
         {
+            if (String.IsNullOrEmpty(action))
+            {
+                Logger.Log(LogLevel.ERROR, Constants.ODP_INVALID_ACTION_MESSAGE);
+                return;
+            }
+
+            if (String.IsNullOrEmpty(type))
+            {
+                type = Constants.ODP_EVENT_TYPE;
+            }
+
```

Comment:
Where do we check if `identifiers` has non-empty?

Reply:
Looks like the check is in OdpEventManager.SendEvent